pkg/apis/wasm/v1alpha1/wasm_module_validation.go

```
@@ -32,24 +33,78 @@ func (as *WasmModule) Validate(ctx context.Context) *apis.FieldError {
+		}
+		for i, addr := range ns.Udp.Outgoing {
+			if err := validateAddressPattern(addr); err != nil {
+				errs = errs.Also(apis.ErrInvalidArrayValue(addr, "udp.outgoing", i))
+			}
+		}
 	}
 
 	return errs
 }
+
+// validateAddressPattern validates address patterns like "host:port", "*:port", "host:*", "*:*"
```

golangci-lint (2.5.0)

[error] Comment should end in a period

(godot)